pkgs/python-utils/default.nix

```diff
@@ -32,19 +32,19 @@ let
       ldLibraryPathConvertWrapper = pkgs.writeShellApplication {
         inherit name;
         text = ''
-          if test "''${REPLIT_RTLD_LOADER:-}" = "1" && test "''${REPLIT_NIX_CHANNEL:-}" != "legacy"
+          if test "''${REPLIT_NIX_CHANNEL:-}" = "legacy"
```

Comment:
there might be 2 channels to use, let me check.

Reply:
Try to go by this as well https://github.com/replit/goval/blob/main/pkg/replenv/modules/modules.go#L682

Reply:
ah TIL, I guess we have gotten lucky that few users have hit that case?

Reply:
https://github.com/replit/nixmodules/pull/408/files#diff-d53b1e014cd069c8da29b3a8029151e8f4727a885547eba3ce4422266bca8eecR43